Plugins/Statistics/GitStatistics/CodeFile.cs

```diff
@@ -105,6 +105,8 @@ private void IncrementLineCountsFromLine(string line)
                 NumberCommentsLines++;
             else if (File.Extension.Equals(".cshtml", StringComparison.OrdinalIgnoreCase) && (line.Contains("@*") && line.Contains("*@")))
                 NumberCommentsLines++;
+            else if (File.Extension.Equals(".m", StringComparison.OrdinalIgnoreCase) && line.StartsWith("%"))
```

Comment:
I haven't used matlab in many years so i'm a bit rusty with its syntax
would `<whitespace> % text` be considered as a comment? if so, it is probably worth while trimming the line before the check

Reply:
yes. Your example would be a valid comment

Reply:
`IncrementLineCountsFromLine` is called in `CountLines` with `line.TrimStart()` so there is no need trim again